ethereum/contracts/SimplifiedMMRVerification.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.8.5;
+
+contract  SimplifiedMMRVerification {
+    function verifyInclusionProof(
+        bytes32 root,
+        bytes32 leafNodeHash,
+        bytes32[] memory restOfThePeaks,
+        bool hasRightBaggedPeak,
```

Comment:
can we remove this bool and just always give the rightBaggedPeak? (ie, if the peak is actually the most right, then rightBaggedPeak = 0x0000000000000000)

Reply:
@musnit Updated